### PR TITLE
Add an option to install nightly build

### LINKED ISSUE #9: Support live repositories (per push tarantool builds)
Now this action only supports so called release repositories. Those repositories contain released tarantool versions (which are marked as `x.y.z-0-gxxxxxxxxx`).

It looks worthful to add so called 'live' repositories, which contain per-push builds. It may be useful for developers, which want to catch possible problems prior to actual tarantool release.

### PR DESCRIPTION
**Pull request by rosik**
```yaml
steps:
  - uses: actions/checkout@v2
  - uses: tarantool/setup-tarantool@v1
    with:
      tarantool-version: '2.6'
      nightly-build: true
```

Close #9

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -7,8 +7,11 @@ import * as io from '@actions/io'
 import * as path from 'path'
 import * as fs from 'fs'
 
+const nightlyBuild =
+  (core.getInput('nightly-build') || 'false').toUpperCase() === 'TRUE'
 const baseUrl =
-  'https://download.tarantool.org/tarantool/release/' +
+  'https://download.tarantool.org/tarantool/' +
+  (nightlyBuild ? '' : 'release/') +
```
Comment (Totktonada) [inline src/main.ts:14]: There is explicit 'live' URL part, if you want.

In the next bullets 2 and 3 are equal, but the latter is more explicit.

1. https://download.tarantool.org/tarantool/release/<...>
2. https://download.tarantool.org/tarantool/live/<...>
3. https://download.tarantool.org/tarantool/<...>

**Review by Totktonada — APPROVED** [review]
I don't see any problems.

**Review by Steap2448 — APPROVED** [review]

File: src/main.ts (hunk 1 above)
Comment (rosik) [reply src/main.ts]: Thanks. Accepted.

Comment (rosik) [reply src/main.ts:14]: The trick didn't work:

```console
$ curl -v https://download.tarantool.org/tarantool/live/1.10/gpgkey

HTTP/1.1 302 Moved Temporarily
Location: https://www.tarantool.io/download/
```

I'll merge without it.

Comment (Totktonada) [reply src/main.ts:14]: That's the bug. Sorry. Okay.

**Pull request merged**